Reqnroll.Generator/Reqnroll.Generator.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="System.CodeDom" Version="8.0.0" />
+    <PackageReference Include="UTF.Unknown" Version="2.6.0" />
```

Comment:
I'm not sure if we like to add a new dependency, but at least we should add it to the nuspec then?

Reply:
The assembly is copied in to the Reqnroll.Tools.MsBuild.Generation package during its publication.
```xml
 <files>
    <file src="build\**\*" target="build" />
    <file src="buildMultiTargeting\**\*" target="buildMultiTargeting" />
    
    <file src="bin\$config$\net462\*.dll" target="tasks\$Reqnroll_FullFramework_Tools_TFM$" />
    <file src="bin\$config$\netstandard2.0\*.dll" target="tasks\$Reqnroll_Core_Tools_TFM$" />
    <file src="bin\$config$\netstandard2.0\*.deps.json" target="tasks\$Reqnroll_Core_Tools_TFM$" />
```

The inclusion of UTFUnknown works without explicitly mentioning it in the nuspec.

Reply:
ow yes indeed that works :)